src/test/java/com/ezylang/evalex/functions/datetime/DateTimeFunctionsTest.java

```diff
@@ -23,17 +23,27 @@
 import com.ezylang.evalex.config.ExpressionConfiguration;
 import com.ezylang.evalex.config.TestConfigurationProvider;
 import com.ezylang.evalex.parser.ParseException;
+import java.time.LocalDate;
 import java.time.ZoneId;
+import java.util.TimeZone;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
 import org.junit.jupiter.params.provider.ValueSource;
 
 class DateTimeFunctionsTest extends BaseEvaluationTest {
 
+  private static final ZoneId DEFAULT_ZONE_ID = ZoneId.of("Europe/Berlin");
+
+  static {
+    // Let the default time-zone be set programmatically to standardize
+    // tests dependent on the default time-zone
+    TimeZone.setDefault(TimeZone.getTimeZone(DEFAULT_ZONE_ID));
```

Comment:
Do not set system defaults in tests.
There is no need for this, if you use the configured time zone and not the system time sone, as shown above.